consultant.gemspec

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+Gem::Specification.new do |s|
+  s.name        = 'consultant'
+  s.version     = '0.0.0'
+  s.date        = '2020-11-24'
+  s.summary     = "A suite of tools for pinpointing inefficiencies\n"\
+    'and measuring and presenting performance improvements'
+  s.description = 'Description TBD'
```

Comment:
We can use the description to give a more in-depth explanation of the tools available